docs/using-openstack-cloud-controller-manager.md

```diff
@@ -173,7 +173,8 @@ Although the openstack-cloud-controller-manager was initially implemented with N
   Deprecated.
 * `internal-lb`
   Determines whether or not to create an internal load balancer (no floating IP) by default. Default: false.
-
+* `cascade-delete`
```

Comment:
shouldn't it be `cascade-delete-lb`, as it is specific to load balancers.??

Reply:
I wasn't aware of this naming convention, most of the other options related to load balancer (create-monitor, monitor-delay , monitor-max-retries etc) don't have `-lb` appended to them.

I am happy to change `cascade-delete` -> `cascade-delete-lb`. Please let me know @adisky @lingxiankong

Reply:
@adisky I don't think `-lb` is needed, all the options here are in the `LoadBalancer` config section, adding `-lb` seems redundant.